apps/server/src/modules/h5p-editor/service/libraryStorage.service.ts

```diff
@@ -282,7 +262,18 @@ export class LibraryStorage implements ILibraryStorage {
 	 * @param file
 	 */
 	public async getFileStats(libraryName: ILibraryName, file: string): Promise<IFileStats> {
-		return this.getFileMetadata(libraryName, file);
+		this.checkFilename(file);
+
+		const head = await this.s3Client.head(this.getS3Key(libraryName, file));
+
+		if (head.LastModified === undefined || head.ContentLength === undefined) {
```

Comment:
see above

Reply:
Are you talking about line 267 or 269?
Declaring a variable for the undefined-check in line 269 breaks typescripts inference and it does not detect that both fields are actually defined after the check